pkgs/development/tools/tabnine/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl }:
+
+let
+  platformSpecific = {
+    x86_64-linux = {
+      sha256 = "caa86a00d0ede8485bf855276ac1b2e411a5e99de612696b137ea3519a8e4967";
+      string = "x86_64-unknown-linux-gnu";
+    };
+    x86_64-darwin = {
+      sha256 = "eb5ceb534629dcf5d57640bf672ea0d94ac02e264209ab3a29e6bda8a44d3b8f";
+      string = "x86_64-apple-darwin";
+    };
+  }.${stdenv.hostPlatform.system} or (throw "Unsupported platform");
```

Comment:
The throw is useful only if someone tries to fetch tabnine.src since the check should be done at the meta.platforms in general. Let's keep it